test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -99,10 +99,12 @@ hw.module @M1<param1: i42>(%clock : i1, %cond : i1, %val : i8) {
   sv.initial {
     // CHECK-NEXT:   if (cond)
     sv.if %cond {
-      %c42 = hw.constant 42 : i42
+      %c42 = hw.constant 42 : i40
 
-      // CHECK-NEXT: wire42 = 42'h2A;
-      sv.bpassign %wire42, %c42 : i42
```

Comment:
Is the logic being tested by the test being removed still being checked elsewhere?  If not, just add a test, rather than repurpose a still in use test.